src/compressed_tensors/quantization/lifecycle/apply.py

```diff
@@ -104,8 +132,92 @@ def load_pretrained_quantization(model: Module, model_name_or_path: str):
             )
 
 
+def process_transforms_config(
+    transforms_config: TransformationConfig,
```

Comment:
Another example where I think consistency in naming convention will help users. Something like this?
```suggestion
    transform_config: TransformConfig,
```